mllib/src/test/scala/org/apache/spark/ml/recommendation/ALSSuite.scala

```diff
@@ -774,6 +774,8 @@ class ALSCleanerSuite extends SparkFunSuite {
     } finally {
       Utils.deleteRecursively(localDir)
       Utils.deleteRecursively(checkpointDir)
+      // Clear the local cache directory
+      Utils.clearLocalRootDirs()
```

Comment:
I am sorry, I had to be clear ... I meant adding `afterEach` and `beforeAll` similarly ...